docs/elasticsearch/ux.md

```diff
@@ -0,0 +1,1240 @@
+{
+  "size": 10000
+}
+```
+
+Elasticsearch caches the results and returns a scroll ID to access them in batches:
+
+```json
+"_scroll_id" : "DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAAUWdmpUZDhnRFBUcWFtV21nMmFwUGJEQQ=="
+```
+
+Pass this scroll ID to the `scroll` operation to get back the next batch of results:
+
+```json
+GET _search/scroll
+{
+  "scroll": "10m",
+  "scroll_id": "DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAAUWdmpUZDhnRFBUcWFtV21nMmFwUGJEQQ=="
+}
+```
+
+Using this scroll ID, you get results in batches of 10000 as long as the search context is still open. Typically, the scroll ID does not change between requests. But it could change, so make sure you always use the latest scroll ID. If the next scroll request is not sent within the set search context, the results are not returned.
+
+If you expect billions of results, use a sliced scroll. Slicing allows you to perform multiple scroll operations for the same request but in parallel.
+Set the ID and the maximum number of slices we can have for this scroll:
+
+```json
+GET shakespeare/_search?scroll=10m
+{
+  "slice": {
+    "id": 0,
+    "max": 10
+  },
+  "query": {
+    "match_all": {}
+  }
+}
+```
+
+You get back 10 results, with a single scroll ID.
+You can have up to 10 IDs.
+Perform the same command with ID equal to 1:
+
+```json
+GET shakespeare/_search?scroll=10m
+{
+  "slice": {
+    "id": 1,
+    "max": 10
+  },
+  "query": {
+    "match_all": {}
+  }
+}
+```
+
+Make sure to close the search context when you’re done scrolling, because it continues consume computing resources until the timeout:
+
+```json
+DELETE _search/scroll/DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAAcWdmpUZDhnRFBUcWFtV21nMmFwUGJEQQ==
+```
+
+#### Sample Response
+
+```json
+{
+  "succeeded": true,
+  "num_freed": 1
+}
+```
+
+To close all open scroll contexts:
+
+```json
+DELETE _search/scroll/_all
+```
+
+The `scroll` operation is a point-in-time search just like a snapshot corresponding to a specific timestamp. Documents added when the search is context is open are not taken into account in the results.
+
+Because open search contexts consume a lot of memory, we suggest you do not use this for frequent user queries.
+If you don't need the search context open, use the `sort` parameter with the `search_after` parameter for scrolling.
+
+## Sort
+
+For any site search, it’s important for your users to be able to sort the results in a way that’s most meaningful to them.
+
+For full-text queries, results are sorted by relevance score by default.
+We can choose to sort the results by any field value in either ascending or descending order.
+
+For example, to sort results by descending order of a `line_id` value:
+
+```json
+GET shakespeare/_search
+{
+  "query": {
+    "term": {
+      "play_name": {
+        "value": "Henry IV"
+      }
+    }
+  },
+  "sort": [
+    {
+      "line_id": {
+        "order": "desc"
+      }
+    }
+  ]
+}
+```
+
+The sort parameter is an array, so you can specify multiple field values.
```

Comment:
multiple field values... in priority order? Does the order matter?

Reply:
Yes, changed to "multiple field values in order of their priority." It's also illustrated in the example below.